en/common/performance.mdown

```diff
@@ -221,6 +261,10 @@ PFQuery *query = [PFUser query];
 var query = PFUser.query()
 query.whereKey("state", containedIn: ["Invited", "Blocked"])
```

Comment:
mind doing a ninja edit while you're here and fixing this to be `notContainedIn`?